### ISSUE TITLE
Recommend registering as a user service instead of a system service in README.md

### ISSUE THREAD

**Issue by machitgarha** [albirar/telegram-bot-echo#1]
Why the bot is registered as a system service, thus requiring the use of `sudo`, which could be harmful? Instead, make it a user service (via `systemctl --user`), which does not require root privileges.

**Comment by albirar** [comment]
Yes, if you want you can install as user only, and no sudo privileges should to be used

**Comment by machitgarha** [comment]
@albirar Right, but I meant `README.md` should recommend adding it as a user service instead.

**Comment by albirar** [comment]
As an alternative to system wide service. I reopen this issue.
Thanks
